PyInstaller/building/makespec.py

```diff
@@ -212,6 +212,10 @@ def __add_options(parser):
     g.add_argument("--noupx", action="store_true", default=False,
                    help="Do not use UPX even if it is available "
                         "(works differently between Windows and *nix)")
+    g.add_argument("--exclude-upx", dest="exclude_upx", metavar="<FILE>", action="append", default=[],
+                   help="Prevent a binary from being compressed using upx."
```

Comment:
"being compressed *when* using upx".

Space missing after period.

Reply:
is somewhat arbitrary linewrap OK here? after correcting the linter error i have this
```
help="Prevent a binary from being compressed when using "
         "upx. This is typically used if upx corrupts certain "
         "binaries during compression. "
         "This option can be used multiple times.")
```
which doesn't read exceptionally well...